### PR TITLE
Update metrics and tracing configs

### PR DESCRIPTION
**Pull request by chickenlj**
followup of #2447

### CODE REVIEW

File: compat.go [hunk 1]
```diff
@@ -273,24 +268,32 @@ func compatMetricConfig(c *global.MetricConfig) *config.MetricConfig {
...
+		Enable:             c.Enable,
+		Port:               c.Port,
+		Path:               c.Path,
+		Prometheus:         compatMetricPrometheusConfig(c.Prometheus),
+		Aggregation:        compatMetricAggregationConfig(c.Aggregation),
+		Protocol:           c.Protocol,
+		EnableMetadata:     c.EnableMetadata,
+		EnableRegistry:     c.EnableRegistry,
+		EnableConfigCenter: c.EnableConfigCenter,
+		EnableRpc:          c.EnableRpc,
```
Comment (FoghostCn) [inline compat.go:280]: Shouldn’t EnableRpc be deleted?

Comment (chickenlj) [reply compat.go:280]: Yeah, but I worry it will introduce new issues. So we can do that in a separate patch later.

Comment (FoghostCn) [reply compat.go:280]: OK

**Review by FoghostCn — APPROVED** [review]
LGTM

**Pull request merged**